DNN Platform/Modules/ResourceManager/Services/ItemsController.cs

```diff
@@ -170,7 +179,7 @@ select new
                         IsDefault =
                         m.MappingName == "Standard" || m.MappingName == "Secure" || m.MappingName == "Database",
                         editUrl = this.UserInfo.IsAdmin ?
-                            moduleContext.EditUrl(
+                            this.GetModuleContext().EditUrl(
```

Comment:
This is going to call `GetModuleContext` for each item in `mappings`, was this change intentional?

Reply:
Thanks for mentioning this, it is not intentional and I'll fix it in the next update